gluecodium/src/main/java/com/here/gluecodium/validator/LimePropertiesValidator.kt

```diff
@@ -63,6 +67,12 @@ internal class LimePropertiesValidator(private val logger: LimeLogger) {
             logger.error(limeProperty, "property in an interface cannot be marked with @Cpp(Ref)")
             result = false
         }
+        if (limeProperty.valueComment.isEmpty()) {
```

Comment:
I wonder whether this is an optimal solution.

If the user does not add any comment to the property, then this still prints a warning.


In the beginning I considered to print the warning/error only when user adds a comment, but it does not use `@value` annotation. It can be easily achieved:
`if (limeProperty.valueComment.isEmpty() && !limeProperty.comment.isEmpty())`

What do you think about it? Do many users want to skip adding any documentation to property altogether?

Reply:
I think this warning make sense for a public property. I.e. if a property is internal (or class/struct which has the property is internal, etc) then it's ok to skip documentation.

Reply:
Implemented the logic according to the comment. Added also unit tests that check the mentioned path.